src/slices.py

```diff
@@ -12,26 +12,26 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[1:2])
 
 # Output the second-to-last element: 9
-print()
+print(a[-2])
 
 # Output the last three elements in the array: [7, 9, 6]
-print()
+print(a[-3:])
 
 # Output the two middle elements in the array: [1, 7]
-print()
+print(a[len(a)//2-1:len(a)//2+1])
```

Comment:
Hey good job at getting a way around this. there was no need to use len(). starting from 2 and stopping at 4 would be a better approach

Reply:
I wanted it to be able to handle an array of any length.